docs/configuration/projectiles.md

```diff
@@ -7,7 +7,7 @@ Configurable fields:
 - `base_damage`: Damage done by the projectile on collision
 - `base_speed`: Travel speed of the projectile
 - `base_size`: Size of the projectile for collision math
-- `player_collision`: Determines if the projectile is removed from game after colliding with a player, default is `true`
+- `pierce`: Determines if the projectile is removed from game after colliding with a player, default is `true`
```

Comment:
I have issues with this naming

1. `pierce` in a context of games can mean a lot of different things (e.g. pierce damage), I feel `collision` or something like it is immediately clear
2. The explanation alone is not clear, what happens if I set this to `true`?

Reply:
Renamed to `remove_on_collision`